repair-service/rest-app/pom.xml

```diff
@@ -17,12 +17,19 @@
             <groupId>com.epam</groupId>
             <artifactId>service</artifactId>
             <version>0.0.1-SNAPSHOT</version>
+            <scope>compile</scope>
         </dependency>
 
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+        </dependency>
```

Comment:
it should be in another PR

Reply:
Comment accepted. Test dependencies removed.